pkg/metrics/collector/netlink_metrics.go

```diff
@@ -108,25 +310,19 @@ func getRequests() ([]*parser.Snapshot, error) {
 	return append(res, res6...), nil
 }
 
-func getRetransmits(snapshots []*parser.Snapshot) uint64 {
-	var retransmits uint64
-	retransmits = 0
-	for _, snap := range snapshots {
-		if snap == nil || snap.TCPInfo == nil {
-			continue
-		}
-		retransmits += uint64(snap.TCPInfo.Retrans)
-	}
-	return retransmits
-}
-
 func (c *netlinkCollector) Update(ch chan<- prometheus.Metric) error {
 	snapshots, err := getRequests()
 	if err != nil {
+		glog.Infof("Could not get requests")
 		return fmt.Errorf("could not get tcp requests: %q", err)
 	}
-	retransmits := getRetransmits(snapshots)
-	ch <- prometheus.MustNewConstMetric(TCPConnectionsDesc, prometheus.GaugeValue, float64(len(snapshots)))
-	ch <- prometheus.MustNewConstMetric(TCPConnectionsDesc, prometheus.GaugeValue, float64(retransmits))
```

Comment:
I think it is still useful to have separate node-level metrics in additional to the pod metrics. Could we keep these?

Reply:
Done!

Reply:
Sg!

Reply:
Good point.
Michael, when you add these back can you also add a comment that the difference between these metrics vs a sum over all pod metrics is that these will include metrics from host network pods + processes on the root ns as well.

Reply:
Sounds good!

Reply:
Great, thanks for keeping this.